app/src/main/java/com/heapixLearn/discovery/logic/contact/LogicContactManager.java

```diff
@@ -0,0 +1,103 @@
+package com.heapixLearn.discovery.logic.contact;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.DBStoreContactManager;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.ServerStoreContactManager;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.UIContactManager;
+
+import java.util.ArrayList;
+
+public class LogicContactManager {
+    private static String TAG = "!!!LOG!!!";
+    static LogicContactManager instance;
+    DBStoreContactManager dbManager;
+    ServerStoreContactManager serverManager;
+    UIContactManager uiManager;
```

Comment:
@dianaprimshits it slightly ruins the architecture which we discussed, this involves using a new component for managing logic listeners